reference/api/README.md

```diff
@@ -14,9 +14,13 @@ Check out [the FAQ](/resources/faq.md) for answers to some common questions 💡
 
 #### Content type
 
-Every request and response body is in `JSON`.
+ Requests can be in JSON, CSV, or NDJSON but responses are always in JSON.
```

Comment:
It's not always pretty, but in contexts like API docs I always prefer to use short and dry sentences.

```suggestion
 Requests can be in JSON, CSV, or NDJSON. Responses are always in JSON.
```